jax/_src/numpy/fft.py

```diff
@@ -268,8 +299,9 @@ def fftshift(x, axes=None):
 
 
 @_wraps(np.fft.ifftshift)
-def ifftshift(x, axes=None):
+def ifftshift(x: ArrayLike, axes: Union[None, int, Sequence[int]] = None):
```

Comment:
Is the omission of return type here intentional?

Reply:
No, not intentional, thanks for catching that!